apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import ProjectCard from './ProjectCard';
+
+const styles = {
+  grid: {
+    padding: 10,
+    width: 1000
+  },
+  card: {
+    display: "inline-block",
+    padding: 10
+  },
+  labHeading: {
+    textAlign: "left",
+    fontSize: 24,
+    color: "#5b6770",
+    marginLeft: 10,
+    marginRight: 65,
+    borderBottom: "1px solid #bbbbbb",
+    paddingBottom: 10,
+    paddingTop: 40
+  }
+};
+
+const ProjectCardGrid = React.createClass({
+  propTypes: {
+    projects: React.PropTypes.array,
+    galleryType: React.PropTypes.string
+  },
+
+// Most recently edited projects should display 1st. This might not be needed dependent on whether the projects returned from the query are already sorted by recency or not. ****ASK DAVE****
+  sortByDate(projects) {
+
+    let sortedProjects = projects.sort((project1, project2) => new Date(project2.projectData.updatedAt) - new Date(project1.projectData.updatedAt));
+
+    return sortedProjects;
+  },
+
+  sortByType(projects) {
+    let applabProjects = [];
+    let gamelabProjects = [];
+    let artistProjects = [];
+    let playlabProjects = [];
+    let weblabProjects = [];
```

Comment:
One thing you could do rather than having a bunch of individual arrays is to just store a single object where the keys are project types, and the values are arrays of projects. Then you could do something like

```
projects.forEach(project => {
  const type = project.projectData.type;
  projectLists[type] = projectLists[type] || [];
  projectLists[type].push(project);
});
```
One benefit of this is that it will make us more resilient to adding new project types in the future.